lib/ui/window/pointer_data.cc

```diff
@@ -8,6 +8,17 @@
 
 namespace flutter {
 
+// The number of fields of PointerData.
+//
+// If kPointerDataFieldCount changes, update the corresponding values to:
+//
+//  * _kPointerDataFieldCount in platform_dispatcher.dart
+//  * POINTER_DATA_FIELD_COUNT in AndroidTouchProcessor.java
```

Comment:
Should this include `PointerData in pointer_data.h`?

Reply:
`PointerData` points to `kPointerDataFieldCount`, not the other way around. You add a field in `PointerData`, you're pointed to `kPointerDataFieldCount`, and you update other fields.

I've added `pointer_data_packet_converter_unittests.cc` in. Thanks for pointing it out.